pkg/controller/pcideviceclaim/pcideviceclaim_controller.go

```diff
@@ -312,7 +267,7 @@ func (h Handler) rebindAfterReboot() error {
 		}
 
 		// Enable Passthrough on the device
-		err = h.enablePassthrough(pd)
+		err = h.enablePassthrough(pd, pdcCopy)
```

Comment:
```suggestion
		err = h.enablePassthrough(pd)
```

updated to reflect the new method signature